locomotion-sdk/src/services/geo.js

```diff
@@ -95,5 +96,21 @@ class Geo {
   prepareCoords = locations => ({ coords: { latitude: locations[0].latitude, longitude: locations[0].longitude }, timestamp: new Date() })
 }
 
-export default new Geo();
+const GeoService = new Geo();
+export default GeoService;
+
 export const { decodeGmPath } = Geo;
+
+export const getPosition = async () => {
+  try {
+    return GeoService.currentLocation();
```

Comment:
try/catch is useless here unless you replace the return with await and return the result after the try/catch.

Reply:
agreed, but just a moving refactor.